lkurashchenko/lab1/2/Makefile

```diff
@@ -0,0 +1,26 @@
+CC =gcc
+CFLAGS=-c -Wall
+CFILE=hello_static.c
+LIB=static.a
+OBJECT=hello.o
+MAIN=hello.c
+
+compile: $(CFILE)
```

Comment:
Лучше придерживаться одного стиля в makefile, т.к. в первом и втором пункте makefile ведет себя по-разному (там ничего не делает, тут компилирует), если вызвать без аргументов, просто `make`
>By default, it begins by processing the first target that does not begin with a . aka [the default goal](https://www.gnu.org/software/make/manual/html_node/How-Make-Works.html); to do that, it may have to process other targets - specifically, ones the first target depends on.

Reply:
добавил all, теперь по умолчанию всё собирает и запускает

Reply:
Вообще, нужно было сделать наоборот
Makefile не предназначен для запуска программ, только для компиляции. Я по умолчанию (`make`) ожидаю, что программа только соберется, а для запуска (если хочется) можно оставить отдельный `make run`
Исправлять уже не нужно, замечание на будущее